docs/querying/api.md

```diff
@@ -936,6 +936,27 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+
+## Remote Read API
+
+This API provides data read functionality from prometheus. This interface expects [snappy](https://github.com/google/snappy) compression.
```

Comment:
```suggestion
This API provides data read functionality from Prometheus. This interface expects [snappy](https://github.com/google/snappy) compression.
```

Reply:
Updated locallly (will appear in next patch)